proposals/src/header.rs

```diff
@@ -47,11 +59,6 @@ pub struct ResourceId(pub [u8; 32]);
         scale_codec::MaxEncodedLen
     )
 )]
-#[cfg_attr(
-    feature = "std",
-    derive(Serialize, Deserialize),
-    serde(tag = "type", content = "id")
-)]
```

Comment:
We still need these serde tags, it will be a breaking change if we removed these attr